css/at-rules/media.json

```diff
@@ -917,7 +917,7 @@
                 "version_added": "1.3"
               },
               "safari_ios": {
-                "version_added": "3.1"
+                "version_added": "1"
```

Comment:
That 1.3 &rarr; 1 mapping here is correct, but are you confident that the Safari data is correct and not the iOS data?

Reply:
I've opened #11188 to fix the inaccurate data here.